pkg/arvo/sys/zuse.hoon

```diff
@@ -6036,6 +6037,57 @@
       ?~  myn  ~
       ?:  =(~ t.myn)  (trip i.myn)
       (weld (trip i.myn) `tape`['/' $(myn t.myn)])
+    ::
+    ::  |base16: en/decode arbitrary MSB-first hex strings
```

Comment:
I went with MSB-first because that's standard for hex, and matches the behavior of our existing use-cases. But the byte-order remains an open question for the other standard encodings I'd like to see here: base32/58/64.

Our implementations have generally only been suitable for cords, and, as such, have tended to use LSB-first order. I see these conversions in zuse as being primarily about interop with external systems (otherwise the auras would work). It really just comes down to which types of input need to be byte-swapped, but it'd be nice to have consistent answers.